test/test_qt_utils.py

```diff
@@ -0,0 +1,223 @@
+"""
+Test Module for qt_utils
+"""
+import numpy as np
+import pytest
+from c3.utils.qt_utils import (
+    pauli_basis,
+    basis,
+    xy_basis,
+    get_basis_matrices,
+    rotation,
+    np_kron_n,
+    hilbert_space_kron,
+    kron_ids,
+    projector,
+    pad_matrix,
+    perfect_parametric_gate,
+    T1_sequence,
+    ramsey_sequence,
+    ramsey_echo_sequence,
+)
+from numpy.testing import assert_array_almost_equal as almost_equal
+from c3.libraries.constants import GATES
+
+
+@pytest.mark.unit
+def test_pauli_basis() -> None:
+    """Testing dimensions of Pauli basis"""
+    dims = np.random.randint(2, 5, np.random.randint(1, 5))
+    result = pauli_basis(dims)
+    assert result.shape[0] == result.shape[1]
+    assert result.shape[0] == dims.prod() ** 2
+
+
+@pytest.mark.unit
+def test_basis() -> None:
+    """Testing orthonormality of basis vectors."""
+    for dim in [3, 5, 10, 100]:
```

Comment:
The choice of `[3, 5, 10, 100]` is used in multiple places throughout this test file. Please update the code to define this list as a global constant at the top of this test file, so that it is possible to alter this at just one point in the future without the need for editing in several locations.

Reply:
Sure, I moved it to a fixture. I also reduced 100 to 50 because some matrix products took very long.